rdmo/accounts/tests/helpers.py

```diff
@@ -0,0 +1,68 @@
+import sys
+from importlib import import_module, reload
+
+import pytest
+
+from django.urls import clear_url_caches, get_resolver, reverse
+
+
+def reload_urlconf(urlconf=None, root_urlconf=None):
```

Comment:
In principle the `reload_urlconf` and `reload_urls` could move to a `core.tests.helpers` but we can keep them here for now and move them if we need them again (elsewhere).